app/templates/DonationForm.html.twig

```diff
@@ -222,6 +222,14 @@
 
         // connect DOM elements to actions
 
+        function formPage2IsValid() {
```

Comment:
The method name could be more descriptive. Maybe something like `donationDataIsValid()` or `donationDataIsComplete()`?